src/viem/getLockableWallet.ts

```diff
@@ -9,13 +9,25 @@ import {
   WalletActions,
   WalletRpcSchema,
   createWalletClient,
-  http,
 } from 'viem'
 import { privateKeyToAccount } from 'viem/accounts'
-import { writeContract } from 'viem/actions'
+import { writeContract, sendTransaction } from 'viem/actions'
+import { viemTransports } from 'src/viem'
+import networkConfig from 'src/web3/networkConfig'
+import { Network } from 'src/transactions/types'
 
 const TAG = 'viem/getLockableWallet'
 
+export function getTransport(chain: Chain): Transport {
```

Comment:
Yes, it's definitely a bit strange.. we have to do a reverse lookup since we create the viem wallet using `Chain`, which is the _value_ in the `networkConfig.viemChain` mapping, but `Network` is used to key the `viemTransports` mapping. We unfortunately can't use `Chain` to key this transport mapping, since it's not a simple type. Yeah -- this should be simpler the viem wallet creation uses `Network`/`NetworkId` instead.